frontend/src/components/EmployeeModal/EmployeeModal.tsx

```diff
@@ -62,10 +67,12 @@ const EmployeeModal = () => {
       })).then(() => refreshDrivers());
     }
     closeModal();
+    setToast(true);
```

Comment:
we dont want this to show if they close out before submitting, only when a submission is successful